index.html

```diff
@@ -506,6 +506,37 @@ <h3>Process pending pointer capture</h3>
                         <li>Set the <dfn>pointer capture target override</dfn> to the <a>pending pointer capture target override</a>, if set. Otherwise, clear the <a>pointer capture target override</a>.</li>
                     </ol>
                 </section>
+
+                <section>
+                    <h3>Suppressing a pointer event stream</h3>
+                    <p>The user agent MUST <dfn>suppress a pointer event stream</dfn> when it detects that a pointer is unlikely to continue to produce events.  Any of the following scenarios satisfy this condition (there MAY be additional scenarios):</p>
+                    <ul>
+                        <li>The user agent has opened a modal dialog or menu.</li>
+                        <li>A pointer input device is physically disconnected, or a hoverable pointer input device (e.g. a hoverable pen/stylus) has left the hover-range detectable by the digitizer.</li>
+                        <li>The pointer is subsequently used by the user agent to manipulate the page viewport (e.g. panning or zooming).  See the section on <a>touch-action</a> CSS property for details.
+                            <div class="note">User agents can trigger panning or zooming through multiple pointer types (such as touch and pen), and therefore the start of a pan or zoom action may result in the suppression of various pointers, including pointers with different pointer types.</div>
+                        </li>
+                        <li>As part of the drag operation initiation algorithm as defined in the <a data-cite="html/#drag-and-drop-processing-model">drag and drop processing model</a> [[HTML]],
+                         for the pointer that caused the drag operation.</li>
+                    </ul>
+                    <div class="note">
+                        <p>Other scenarios in which the user agent MIGHT <a>suppress a pointer event stream</a> include:
+                        <ul>
+                            <li>A device's screen orientation is changed while a pointer is active.</li>
+                            <li>The user inputs a greater number of simultaneous pointers than is supported by the device.</li>
```

Comment:
`The user inputs a greater number of simultaneous pointers` maybe tweak a bit to `The user attempts to interact with the content using more simultaneous pointers`